news-bundle/src/Resources/contao/dca/tl_news.php

```diff
@@ -81,8 +80,11 @@
 			'fields'                  => array('date'),
 			'headerFields'            => array('title', 'jumpTo', 'tstamp', 'protected', 'allowComments'),
 			'panelLayout'             => 'filter;sort,search,limit',
-			'child_record_callback'   => array('tl_news', 'listNewsArticles'),
-			'child_record_class'      => 'no_padding'
```

Comment:
_Note:_ this seems to be a remnant of old Contao versions, as such a CSS class does not exist anymore.